src/layouts/MainLayout/components/Sidebar/components/ArticleLink/ArticleLink.js

```diff
@@ -1,9 +1,11 @@
 import { useContext } from 'react';
 import { Link, useNavigate, useParams } from 'react-router-dom';
+import { createConfirmation } from 'react-confirm';
 import classNames from 'classnames';
 import { IoIosCreate, IoMdTrash } from 'react-icons/io';
 
 import { ArticlesContext } from '../../../../../../pages/article/context';
+import { ConfirmWindow } from '../../../../../../popups';
```

Comment:
No-no-no, `popup` is just a component, no need in the root-placed folder here :)